receiver/k8sobjectsreceiver/receiver.go

```diff
@@ -50,6 +51,10 @@ func newReceiver(params receiver.Settings, config *Config, consumer consumer.Log
 		return nil, err
 	}
 
+	if err := config.Validate(); err != nil {
+		return nil, err
+	}
```

Comment:
Please remove this, this is incorrect. Validation happens through ConfigValidator casting outside of the creation of the receiver.